scripts/get-members.ts

```diff
@@ -0,0 +1,33 @@
+import { db } from './firebase';
+
+let getNetId = (email: string): string => {
+  let pos = email.search('@');
+  return email.slice(0, pos);
+};
+
+const fs = require('fs');
+const dirPath: string = 'scripts/members/';
+fs.mkdirSync(dirPath, { recursive: true });
+
+const jsonFilesList: string[] = fs.readdirSync(dirPath);
+jsonFilesList.forEach((json) =>
+  fs.unlink(dirPath + json, (err) => {
+    if (err) {
+      console.log(err);
+    }
+  })
+);
+
+db.collection('members')
+  .get()
+  .then((memberRefs) => {
+    memberRefs.forEach((memberRef) => {
```

Comment:
what are the memberRefs, and just in general what are references? Do they have something to do with firebase?

Reply:
Yeah pretty much, I'm pretty sure they're just references to documents in firebase